src/test/java/uk/gov/hmcts/reform/cosapi/edgecase/model/CaseDataTest.java

```diff
@@ -38,7 +38,7 @@ class CaseDataTest {
 
     @BeforeEach
     void setUp() {
-        MockitoAnnotations.initMocks(this);
+        MockitoAnnotations.openMocks(this);
```

Comment:
Any specific reason for updating this?

Reply:
Upstream changes from ISDB-244

Reply:
If we are changing method to openMocks then it should be done across all the test classes